httpbakery/discharge.go

```diff
@@ -71,7 +60,6 @@ func AddDischargeHandler(mux *http.ServeMux, rootPath string, svc *bakery.Servic
 		checker: checker,
 	}
 	mux.Handle(path.Join(rootPath, "discharge"), handleJSON(d.serveDischarge))
-	mux.Handle(path.Join(rootPath, "create"), handleJSON(d.serveCreate))
```

Comment:
This was a little concerning to me, but perfectly fine if you've been using the bakery all along on client and server-side. I suppose someone could handle web API compatibility in your services if they really depended on this (which seems a remote possibility). Good move I think.